docs/index.md

```diff
@@ -60,7 +60,7 @@ steampipe plugin install azuread
 | Item        | Description                                                                                                                                                                                                             |
 | ----------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
 | Credentials | Use the `az login` command to setup your [Azure AD Default Connection](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli)                                                                               |
-| Permissions | Grant the following permissions to your user: <br /> <blockquote> <li>Application.Read.All </li><li>  Policy.Read.ConditionalAccess </li><li>  Directory.Read.All </li><li> Domain.Read.All </li><li> Group.Read.All </li><li> IdentityProvider.Read.All </li><li> User.Read.All  </blockquote>                                                                                                                                                                |
+| Permissions | Grant the following permissions to your user: <br /><li> `Application.Read.All` </li><li> `Policy.Read.ConditionalAccess` </li><li> `Directory.Read.All` </li><li> `Domain.Read.All` </li><li> `Group.Read.All` </li><li> `IdentityProvider.Read.All` </li><li> `User.Read.All`                                                                                                                                                              |
```

Comment:
@misraved Can you please put these items in alphabetical order for better readability?